src/ralph/admin/templates/admin/login.html

```diff
@@ -17,42 +17,50 @@
 {% block breadcrumbs %}{% endblock %}
 
 {% block content %}
-  {% if form.errors and not form.non_field_errors %}
-    <p class="errornote">
-    {% if form.errors.items|length == 1 %}{% trans "Please correct the error below." %}{% else %}{% trans "Please correct the errors below." %}{% endif %}
-    </p>
-  {% endif %}
-
-  {% if form.non_field_errors %}
-    {% for error in form.non_field_errors %}
-      <p class="errornote">
-        {{ error }}
-      </p>
-    {% endfor %}
-  {% endif %}
+{% if form.errors and not form.non_field_errors %}
+<p class="errornote">
+  {% if form.errors.items|length == 1 %}{% trans "Please correct the error below." %}{% else %}{% trans "Please correct the errors below." %}{% endif %}
+</p>
+{% endif %}
+
+{% if form.non_field_errors %}
+{% for error in form.non_field_errors %}
+<p class="errornote">
+  {{ error }}
+</p>
+{% endfor %}
+{% endif %}
 
 <div id="content-main">
-  <form action="{{ app_path }}" method="post" id="login-form">{% csrf_token %}
-    <fieldset>
-      <legend>Please login</legend>
-      {{ form.username.errors }}
-      <label>{{ form.username.label }}
-        {{ form.username }}
-      </label>
-      {{ form.password.errors }}
-      <label>{{ form.password.label }}
-        {{ form.password }}
-      </label>
-      <button>{% trans 'Log in' %}</button>
-    </fieldset>
-    <input type="hidden" name="next" value="{{ next }}" />
-    <div class="password-reset-link">
-    {% url 'admin_password_reset' as password_reset_url %}
-    {% if password_reset_url %}
-      <a href="{{ password_reset_url }}">{% trans 'Forgotten your password or username?' %}</a>
-    {% endif %}
+  <div class="row">
+  <div class="large-3 medium-3 small-5 small-centered columns">
+  <center>
```

Comment:
it's deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center